### ISSUE TITLE
Improvement: Make array format configurable and/or default to table format

### ISSUE THREAD

**Issue by geoffturk** [json-editor/json-editor#837]
### General information

- json-editor version: _2.4.0_

I am aiming to use JSON Editor without needing to include properties related to formatting within the JSON Schema instance (for my use case the instance needs to contain only data, not information about how to display that data), but in order to get array input to display in JSON Editor, it must have the key/value pair `"format": "table"` included in the instance.

### Actual behavior

When you leave it out, you get this (note the missing input field, row label and button):

![image](https://user-images.githubusercontent.com/32190662/95734051-47377d00-0c83-11eb-9a31-085784b10f68.png)

### Expected behavior

But when you add the `"format": "table"` key/value pair it displays correctly:

![image](https://user-images.githubusercontent.com/32190662/95734142-68986900-0c83-11eb-8017-9b7a2a9a8300.png)

### Steps to reproduce the behavior

You can try this yourself by going to the interactive editor:  
https://json-editor.github.io/json-editor/

Then scroll down to replace the default schema instance with the instance below, then click Update Schema. Reload the page and try it with and without the `format` key/value pair included.

```json
{
    "type": "object",
    "title": "Test of Array without Format key",
    "properties": {
        "linked_schemas": {
            "title": "Linked Schemas",
            "description": "The schemaIds against which the profile must be validated",
            "type": "array",
            "format": "table",
            "items": {
                "type": "string",
                "pattern": "[A-Fa-f0-9]{64}"
            },
            "minItems": 1,
            "maxItems": 10,
            "uniqueItems": true
        }
    }
}
```

### Possible solutions

There are two possible solutions I can think of to fix/improve this:

1. Automatically assume `format` is `table` for an array as a default if no `format` property is specified.
2. Add an `array_layout` configuration option, as is currently available for objects with `object_layout` (as noted in the README).

I took a look at the code and could not figure out where `format: table` for arrays was recognized and then processed, but if you can point me in the right direction, I would be glad to try to implement one of the above two solutions and create a PR, presuming you are happy to make this type of a change. I do think offering an option to remove a formatting property within a schema instance is a worthwhile improvement.

Thanks in advance for considering this suggestion.

**Comment by germanbisurgi** [comment]
the Problem lays on the `"uniqueItems": true`. This property is used together with enums. see [readme](https://github.com/json-editor/json-editor#arrays)

**Comment by geoffturk** [comment]
@germanbisurgi Thanks for pointing me to the right location. I made a change to the `arrayOfStrings` resolver and updated one test that broke as a result of the change. 

https://github.com/geoffturk/json-editor/commit/b5391ea727887e6248b291f46feb7b69ccd4c376

My issue is now fixed as a result.

Should I create a PR or is there something else you would like me to do before submitting it?

**Comment by germanbisurgi** [comment]
Pr wellcome! :) Then we can do a review

**Comment by germanbisurgi** [comment]
The issue was resolved in #843. Can you try with the new release?

**Comment by geoffturk** [comment]
Tried it and the issue is indeed resolved. Thank you!
